src/main/java/se/sundsvall/snailmail/api/model/SendSnailMailRequest.java

```diff
@@ -52,7 +52,7 @@ public static class Attachment {
 		@NotBlank
 		private String content;
```

Comment:
Is this base64-content?

There is a ValidBase64-annotation available in dept44-common-validators (if you want to add validation for this?)